dotcom-rendering/scripts/webpack/frontend.js

```diff
@@ -22,16 +23,20 @@ const commonConfigs = ({ platform }) => ({
 			: 'eval-cheap-module-source-map',
 	resolve: {
 		alias: {
-			'@root': path.resolve(__dirname, '.'),
-			'@frontend': path.resolve(__dirname, 'src'),
+			// '@root': path.resolve(__dirname, '.'),
+			// '@frontend': path.resolve(__dirname, 'src'),
 			react: 'preact/compat',
 			'react-dom/test-utils': 'preact/test-utils',
 			'react-dom': 'preact/compat',
 		},
+		plugins: [new TsconfigPathsPlugin()],
 		extensions: ['.js', '.ts', '.tsx', '.jsx'],
 		symlinks: false,
 	},
 	plugins: [
+		new webpack.ProvidePlugin({
```

Comment:
`esbuild-loader` was struggling with files using `jsx` where React was not imported, the `webpack.ProvidePlugin` resolves that issue.